motr/init.c

```diff
@@ -32,6 +32,11 @@
 #include "net/net.h"
 #include "net/bulk_emulation/mem_xprt.h"
 #include "net/lnet/lnet.h"
+#ifndef __KERNEL__
+#ifdef ENABLE_LIBFAB
+#  include "net/libfab/libfab.h"
+#endif /*ENABLE_LIBFAB */
+#endif
```

Comment:
The proper way to do this  is to put #ifdef __KERNEL__ and #ifdef ENABLE_LIBFAB in libfab.h, so that you do not have to protect every include with #ifdef-s.

Reply:
Done.